app/views/feedbacks/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+
+<%= form_with model: @feedback, local: true, html: { class: 'max-w-prose mx-auto px-4' } do |form| %>
+  <% if @feedback.errors.any? %>
+
+    <div class="mb-4">
+      <h2 class="text-red-700 text-lg">
+        Please fix <%= pluralize(@feedback.errors.count, "error") %>
+      </h2>
+
+      <ul>
+        <% @feedback.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label "overall_rating", class: "label label-text text-lg " %>
+    <%= form.number_field :overall_rating, class: "input input-bordered input-sm" %>
```

Comment:
make this required. Also we need to say it is out of `100` (I am also fine if we do out of 10 for simplicity, but when we get the number we multiple by 10 to make it out of 100 when stored.